imap_processing/glows/l0/glows_l0_data.py

```diff
@@ -1,4 +1,27 @@
-class HistogramL0:
+from dataclasses import dataclass, fields
+
+from imap_processing.ccsds.ccsds_data import CcsdsData
+
+
+@dataclass
+class GlowsL0:
+    """Data structure for common values across histogram and direct events data.
+
+    Attributes
+    ----------
+    ground_sw_version: str
+        Ground software version
+    packet_file_name: str
+        File name of the source packet
+    """
+
+    ground_sw_version: str
+    packet_file_name: str
+    ccsds_header: CcsdsData
+
+
+@dataclass
+class HistogramL0(GlowsL0):
     """Data structure for storing GLOWS histogram packet data.
 
     Parameters
```

Comment:
`__init__` function's `Parameters` has updated doc strings. Should we remove this or keep it?

Reply:
Good catch, I removed the `Parameters` section from the class and left it in the `__init__`. Hopefully that's the correct structure, I think that's correct according to our autodocs.